en/companion_computer/holybro_pixhawk_jetson_baseboard.md

```diff
@@ -0,0 +1,1177 @@
+
+```sh
+param set UXRCE_DDS_DOM_ID 0
+param set UXRCE_DDS_PTCFG 0
+param set UXRCE_DDS_SYNCC 0
+param set UXRCE_DDS_SYNCT 1
+```
+
+:::
+
+Then reboot the Pixhawk.
+
+#### Ethernet Setup
+
+As covered in the MAVLink section, there is an internal Ethernet switch linking the Jetson and Pixhawk.
+We configured both boards to use the same subnet above.
+However we need to disable MAVLink on the `Ethernet` port and enable `XRCE-DDS`.
+
+You can [modify the parameters](../advanced_config/parameters.md) in QGroundControl parameter editor, or using `param set` in the [MAVLink sheel](../debug/mavlink_shell.md).
+The code below assumes you're commands in the MAVLink shell.
+
+```sh
+param set MAV_2_CONFIG 0  # Disable MAVLINK on Ethernet (so Ethernet can be used for XRCE-DDS)
+param set UXRCE_DDS_CFG 1000 # Ethernet
+param set UXRCE_DDS_PRT 8888  # Set port to 8888
+```
+
+#### Checking the XRCE-DDS Client is Running
+
+::: tip
+The client should automatically start on boot!
+:::
+
+We can check that the client is running using the following command in the MAVLink shell:
+
+```sh
+uxrce_dds_client status
+```
+
+A healthy output should show:
+
+![uXRCE-DDS Client serial status](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/uxrce_dds_serial_agent_running.png)
+
+Also another way to check if the client has started running at the boot is to get `dmesg` output from MAVLINK shell.
+The below output as a part of `dmesg` log mentions the baudrate and the instance (`/dev/tty/S4` is equal to `TELEM2` for Holybro Pixhawk 6X and 6X Pro) the client is running on.
+
+```sh
+Starting UXRCE-DDS Client on /dev/ttyS4
+INFO  [uxrce_dds_client] init serial /dev/ttyS4 @ 921600 baud
+```
+
+### XRCE-DDS Agent & ROS 2 Setup
+
+Follow the instruction in the ROS2 User guide to [Install ROS 2](../ros2/user_guide.md#install-ros-2) on the Jetson.
+You don't have to install PX4 on the Jetson because we're not using the simulator, but you may wish to install the full desktop so you can use additional ROS 2 packages for further development:
+
+```sh
+sudo apt install -y ros-humble-desktop-full -y
+```
+
+#### Start the XRCE-DDS Agent
+
+Then fetch and build the uXRCE-DDS _agent_ as covered in [Setup the Agent](../ros2/user_guide.md#setup-the-agent) in the ROS 2 User guide.
+
+You will start the agent using different commands, depending on whether the client is configured for a serial or Ethernet connection.
+Assuming the client is set up as defined above:
+
+- Start the agent on `/dev/ttyTHS1` (serial connection):
+
+  ```sh
+  sudo MicroXRCEAgent serial --dev /dev/ttyTHS1 -b 921600
+  ```
+
+- Start the agent on UDP port `8888` (Ethernet):
+
+  ```sh
+  MicroXRCEAgent udp4 -p 8888
+  ```
+
+If your agent and client are running you should see output similar to that below:
+
+![uXRCE-DDS Agent Health check](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/xrce_dds_healthy_connection.png)
```

Comment:
This has changed - it is what the agent sees if the example isn't running. Can you make this into text and replace the image?